tests/api.c

```diff
@@ -33105,6 +33111,78 @@ static int test_wolfSSL_PEM_PrivateKey(void)
     return 0;
 }
 
+static int test_wolfSSL_PEM_file_RSAKey(void)
+{
+#if (defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL)) && \
+    defined(WOLFSSL_KEY_GEN) && !defined(NO_RSA) && \
+    !defined(HAVE_USER_RSA) && !defined(NO_FILESYSTEM) && !defined(NO_CERTS)
+    RSA* rsa = NULL;
+    XFILE fp;
+
+    printf(testingFmt, "wolfSSL_PEM_file_RSAKey");
+
+    AssertTrue((fp = XFOPEN("./certs/rsa-pub-2048.pem", "rb")) != XBADFILE);
```

Comment:
Not necessary to change this now, but in the future, if we want the ability to run all tests without aborting on the first failure, we need to stop using the Assert* family of functions. Tests should signal an error by returning a negative value. Just leaving this comment here for posterity.

Reply:
*sigh*
Yes, the whole thing needs to be reworked!
And you promised! ;-)

Reply:
I pray for whomever takes on refactoring every test. :grimacing: